testdata/test_filter_by_comm.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+. $(dirname "$0")/common.sh
+set -ex
+
+CMD="$1"
+FILE_PREFIX="/tmp/kyanos"
+BEFORE_LNAME="${FILE_PREFIX}_filter_by_comm_before.log"
+AFTER_LNAME="${FILE_PREFIX}_filter_by_comm_after.log"
+
+function test_filter_by_comm() {
+    # server start before kyanos
+    timeout 40 python3 ./testdata/start_http_server.py  &
+    timeout 30 ${CMD} watch --debug-output http --comm python3 2>&1 | tee "${BEFORE_LNAME}" &
+    sleep 10
+    curl http://127.0.0.1:8080 &>/dev/null || true
```

Comment:
You can use testdata/https-request/http_request.go to request the server in a long connection mode. Currently, the short connection is not very stable and still needs to be optimized.